sdks/Make.config.sample

```diff
@@ -1,22 +0,0 @@
-#This is a sample file of all options that can be used by a Make.config file.
```

Comment:
Why are we deleting this file?

Reply:
That was part of your initial commit.

Reply:
err, then there's something wrong with it.

Reply:
Ok, let me revert it then.